src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewSpaceCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors /
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import UIKit
+
+class DynamicTableViewSpaceCell: UITableViewCell {
```

Comment:
To be honest, using a cell to add some spacing between cells seems a bit overkill. But maybe it is just me.

Using a cell purely for layout related things means that it will always be some kind of special case. This may not be an issue now but imagine the impact a spacing cell has on anything related to `IndexPath`s: Selection handling, gesture handling (swipe to delete, …). Also the space cell is only used once as far I can tell. Have you considered simply increasing the padding/margin of the body cell? Wouldn't that have the same effect?

Reply:
Since the point of the `DynamicTableViewController` is to allow a declarative design approach this cell may have some benefit. Alternatively one has to split semantically linked sections into multiple to have the desired spacing between cells.

One has two be careful about the index path with this controller anyway, for the exact same reason of the declarative design approach.